update-dependencies/Program.cs

```diff
@@ -3,152 +3,138 @@
     {
-        private const string CliBuildInfoName = "Cli";
-        private const string SharedFrameworkBuildInfoName = "SharedFramework";
-        private static readonly string s_repoRoot = Directory.GetCurrentDirectory();
-        private static readonly Config s_config = Config.s_Instance;
-        private static bool s_updateOnly = false;
+        private static Options Options { get; set; } = new Options();
+        private static string RepoRoot { get; set; } = Directory.GetCurrentDirectory();
+        private const string RuntimeBuildInfoName = "Runtime";
+        private const string SdkBuildInfoName = "Sdk";
 
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
             try
             {
                 Trace.Listeners.Add(new TextWriterTraceListener(Console.Out));
 
-                if (ParseArgs(args))
-                {
-                    DependencyUpdateResults updateResults = UpdateFiles();
+                Options.Parse(args);
 
-                    if (!s_updateOnly && updateResults.ChangesDetected())
+                DependencyUpdateResults updateResults = await UpdateFiles();
+                if (updateResults.ChangesDetected())
+                {
+                    if (Options.UpdateOnly)
                     {
-                        CreatePullRequest(updateResults).Wait();
+                        Trace.TraceInformation($"Changes made but no GitHub credentials specified, skipping PR creation");
+                    }
+                    else
+                    {
+                        await CreatePullRequest(updateResults);
                     }
                 }
             }
             catch (Exception e)
             {
                 Console.Error.WriteLine($"Failed to update dependencies:{Environment.NewLine}{e.ToString()}");
+                Environment.Exit(1);
             }
-        }
 
-        private static bool ParseArgs(string[] args)
-        {
-            foreach (string arg in args)
-            {
-                if (string.Equals(arg, "--Update", StringComparison.OrdinalIgnoreCase))
-                {
-                    s_updateOnly = true;
-                }
-                else if (arg.Contains('='))
-                {
-                    int delimiterIndex = arg.IndexOf('=');
-                    string name = arg.Substring(0, delimiterIndex);
-                    string value = arg.Substring(delimiterIndex + 1);
-
-                    Environment.SetEnvironmentVariable(name, value);
-                }
-                else
-                {
-                    Console.Error.WriteLine($"Unrecognized argument '{arg}'");
-                    return false;
-                }
-            }
-
-            return true;
+            Environment.Exit(0);
         }
 
-        private static DependencyUpdateResults UpdateFiles()
+        private static async Task<DependencyUpdateResults> UpdateFiles()
```

Comment:
`async` methods missing `Async` suffix in names.

Reply:
Renamed